dynd/nd/array.pyx

```diff
@@ -48,6 +55,8 @@ cdef extern from 'array_functions.hpp' namespace 'pydynd':
     object array_index(_array&) except +translate_exception
     object array_nonzero(_array&) except +translate_exception
 
+    void init_assign() except +translate_exception
```

Comment:
The wrapper for this is already defined on line 30. I'm not sure why having a double declaration for this doesn't cause an error.